src/wovnio/utils/request_handlers/CurlRequestHandler.php

```diff
@@ -64,6 +66,6 @@ protected function post($url, $request_headers, $data, $timeout)
 
         curl_close($curl_session);
 
-        return array($response_body, $headers, null);
+        return array($response_body, $parsedHeaders, null);
```

Comment:
So $headers was previously not being used before and that's why this is the only change. What was the format of $headers before?

Reply:
$headers before was just the header split by newlines, as an array. $headers was used internally for logs, but the returned value was not really used by anyone. The new format actually attempts to parse them into key-value pairs as an associative array.